### PR TITLE
[oneDNN][CPU] fuse a matmul pattern

### PR DESCRIPTION
**Pull request by gaurides**
This PR helps fuse this sub-graph MatMul + BiasAdd + Mul + Add + Elu.
Mul & Add come from BatchNorm and they can be folded into Matmul + BiasAdd to create MM + BiasAdd + Elu which can be easily fused into _FusedMatMul which has an optimized implementation on cpu using oneDNN. With this PR we see upto 18% perf gain.

Pattern before fusion:
image

After folding and fusion:
image

### CODE REVIEW

**Comment by keerthanakadiri** [comment]
Hi @ezhulenev , Can you please review this PR? Thank you !

File: tensorflow/core/grappler/optimizers/remapper.cc [hunk 1]
```diff
@@ -1914,6 +1915,66 @@ bool FindMulAndMaximum(RemapperContext* ctx, int node_index,
...
+                }
+              },
+              {"*", "mul_in2", NodeStatus::kRemain}
+            }
+          },
+          {"*", "add_in2", NodeStatus::kRemain}
+        }
+      }
+    }
+    };
```
Comment (cantonios) [inline tensorflow/core/grappler/optimizers/remapper.cc]: nit: this final bracket is indented too much.

Why disable clang-format?

**Review by cantonios — CHANGES_REQUESTED** [review]

File: tensorflow/core/grappler/optimizers/remapper.cc (hunk 1 above)
Comment (gaurides) [reply tensorflow/core/grappler/optimizers/remapper.cc]: fixed the final bracket indentation. 
clang-format doesn't like the way it is formatted currently. it tries to combine two braces in one row etc making it difficult to read.

**Review by cantonios — APPROVED** [review]
